scripts/partition_manager_report.py

```diff
@@ -41,11 +42,13 @@ def print_region(region, size, pm_config):
     sorted_pm_config = sorted(sorted_pm_config, key=lambda x: pm_config[x]['address'])
 
     # Create text lines
-    lines = ['{}{}: {} ({}){}'.format(
+    lines = ['{}{}: {} ({} - {}{}){}'.format(
                 '| '+bcolors['WARNING'] if 'span' not in pm_config[name] else '+---'+bcolors['OKBLUE'],
                 hex(pm_config[name]['address']),
                 name,
                 hex(pm_config[name]['size']),
+                pm_config[name]['size'] if pm_config[name]['size'] < 1024 else int(pm_config[name]['size']/1024),
```

Comment:
If you use a lambda here, then you will signifficantly shorten the line:
```suggestion
               (lambda x: x if x < 1024 else int(x/1024))(pm_config[name]['size']),
```